packages/esbuild/package.json

```diff
@@ -57,7 +57,7 @@
   "scripts": {
     "prepack": "npm run build",
     "build": "tsc --build --clean && tsc --build && type-coverage",
-    "test-api": "uvu test \"\\.js$\"",
+    "test-api": "node test/index.test.js",
```

Comment:
The extra `.test` is superfluous to as all test files are already in a test folder. And it’s not how the rest of the projects look like.

Reply:
I agree. I decided not to touch rename the file for now. I could though.

Reply:
Ahh okay missed that, yeah, uvu needed the file structure. I don’t mind renaming now!